lsp-bridge-ref.el

```diff
@@ -181,6 +195,9 @@ used to restore window configuration after finish search.")
   "Save buffer point before search,
 used to restore buffer point after finish search.")
 
+(defvar lsp-bridge-ref-requst-search-window nil
```

Comment:
Should ```lsp-bridge-ref-request-search-window``` ? I found miss `e` in `request`.

Reply:
Fixed